examples/MODULE.bazel

```diff
@@ -5,18 +5,26 @@ bazel_dep(name = "rules_zig", version = "20240912.0-41bfe84")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "zml", version = "0.1.0")
 bazel_dep(name = "aspect_bazel_lib", version = "2.8.1.1")
-
 bazel_dep(name = "rules_oci", version = "2.0.0")
+
 oci = use_extension("@rules_oci//oci:extensions.bzl", "oci")
+
 oci.pull(
     name = "distroless_cc_debian12",
-    digest = "sha256:1850aee2ff72864350058d83d681c757d45c885986d15fcca7309b9e5c69f39a",
+    digest = "sha256:b6e1e913f633495eeb80a41e03de1a41aa863e9b19902309b180ffdc4b99db2c",
```

Comment:
can you explain what's going on here ?

Reply:
Since I added a platform, the sha changes

Reply:
what does that mean that an image supports two platform ? if there are two images underlying why isn't there two shas ?

sorry this is all new to me.

Reply:
The digest passed to `oci_pull` is actually an identifier that is used to download a `manifest` which can either be:

1. an `index` in case of multi platform, referencing digests for each individual platform images `manifest`.
2. an image `manifest` in case of single platform, referencing digests for its layers.

The new SHA was referencing the `index` manifest containing the digests of the manifests for the various supported platforms (`linux/amd64`, `linux/arm64/v8`)

See:
https://gcr.io/v2/distroless/cc-debian12/manifests/sha256:b6e1e913f633495eeb80a41e03de1a41aa863e9b19902309b180ffdc4b99db2c

The old SHA was referencing the `manifest` of the `linux/amd64` image directly.

Anyway, I wrongly assumed that arm64 was supported and I closed this PR since it would be a bug to use the arm64 image here.